libweasyl/alembic/versions/5a073c92aab9_deferrable_foreign_keys_revisited_.py

```diff
@@ -0,0 +1,447 @@
+    op.drop_constraint('blocktag_userid_fkey', 'blocktag', type_='foreignkey')
+    op.drop_constraint('blocktag_tagid_fkey', 'blocktag', type_='foreignkey')
+    op.create_foreign_key('blocktag_userid_fkey', 'blocktag', 'login', ['userid'], ['userid'], onupdate='CASCADE', ondelete='CASCADE', deferrable=True)
+    op.create_foreign_key('blocktag_tagid_fkey', 'blocktag', 'searchtag', ['tagid'], ['tagid'], deferrable=True)
+    op.drop_constraint('fursona_userid_fkey', 'character', type_='foreignkey')
+    op.create_foreign_key('character_userid_fkey', 'character', 'login', ['userid'], ['userid'], deferrable=True)
+    op.drop_constraint('charcomment_targetid_fkey', 'charcomment', type_='foreignkey')
+    op.drop_constraint('charcomment_hidden_by_fkey', 'charcomment', type_='foreignkey')
+    op.drop_constraint('charcomment_userid_fkey', 'charcomment', type_='foreignkey')
+    op.create_foreign_key('charcomment_hidden_by_fkey', 'charcomment', 'login', ['hidden_by'], ['userid'], ondelete='SET NULL', deferrable=True)
+    op.create_foreign_key('charcomment_targetid_fkey', 'charcomment', 'character', ['targetid'], ['charid'], deferrable=True)
+    op.create_foreign_key('charcomment_userid_fkey', 'charcomment', 'login', ['userid'], ['userid'], deferrable=True)
+    op.drop_constraint('collection_submitid_fkey', 'collection', type_='foreignkey')
+    op.drop_constraint('collection_userid_fkey', 'collection', type_='foreignkey')
+    op.create_foreign_key('collection_submitid_fkey', 'collection', 'submission', ['submitid'], ['submitid'], deferrable=True)
+    op.create_foreign_key('collection_userid_fkey', 'collection', 'login', ['userid'], ['userid'], onupdate='CASCADE', ondelete='CASCADE', deferrable=True)
+    op.drop_constraint('comments_target_user_fkey', 'comments', type_='foreignkey')
+    op.drop_constraint('comments_hidden_by_fkey', 'comments', type_='foreignkey')
+    op.drop_constraint('comments_target_sub_fkey', 'comments', type_='foreignkey')
+    op.drop_constraint('comments_parentid_fkey', 'comments', type_='foreignkey')
+    op.drop_constraint('comments_userid_fkey', 'comments', type_='foreignkey')
+    op.create_foreign_key('comments_userid_fkey', 'comments', 'login', ['userid'], ['userid'], deferrable=True)
+    op.create_foreign_key('comments_hidden_by_fkey', 'comments', 'login', ['hidden_by'], ['userid'], ondelete='SET NULL', deferrable=True)
+    op.create_foreign_key('comments_target_sub_fkey', 'comments', 'submission', ['target_sub'], ['submitid'], deferrable=True)
+    op.create_foreign_key('comments_target_user_fkey', 'comments', 'login', ['target_user'], ['userid'], deferrable=True)
+    op.create_foreign_key('comments_parentid_fkey', 'comments', 'comments', ['parentid'], ['commentid'], deferrable=True)
+    op.drop_constraint('commishclass_userid_fkey', 'commishclass', type_='foreignkey')
+    op.create_foreign_key('commishclass_userid_fkey', 'commishclass', 'login', ['userid'], ['userid'], deferrable=True)
```

Comment:
Should this have `onupdate='CASCADE', ondelete='CASCADE'` to match the table definition?